cdn/signed-urls/src/main/java/com/google/cdn/SignedUrls.java

```diff
@@ -33,6 +33,7 @@
 public class SignedUrls {
 
   // [START signUrl]
```

Comment:
Please remember to remove this one once you transition the docs to the new region_tag.